pkg/storage/cedar_system_bookmark.go

```diff
@@ -9,16 +9,18 @@ import (
 
 	"github.com/cmsgov/easi-app/pkg/appcontext"
 	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/helpers"
 	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlqueries"
 )
 
 // CreateCedarSystemBookmark creates a new cedar system bookmark object in the database
 func (s *Store) CreateCedarSystemBookmark(ctx context.Context, cedarSystemBookmark *models.CedarSystemBookmark) (*models.CedarSystemBookmark, error) {
-	euaUserID := appcontext.Principal(ctx).ID()
-	createAt := s.clock.Now().UTC()
+	if len(cedarSystemBookmark.EUAUserID) < 1 {
+		cedarSystemBookmark.EUAUserID = appcontext.Principal(ctx).ID()
+	}
```

Comment:
Are there instances where we don't pass the EUA User ID? Could we just require it to be set in the model every time?

Reply:
That is something I am debating - I am not sure if we should even allow the passing of an EUA here. I think it should always be derived, the data type though as a retrieval needs it

Reply:
But as I say that, maybe the retrieval doesn't need it either - it just needs to be used in the query

Reply:
@ClayBenson94 i don't think i actually answered your question directly - there were not really instances where we weren't passing the EUA, but we still re-set it in the store method using `ID()`

Reply:
I'm fine with any approach, really -- I only suggest it because sometimes it's nice (like, for testing) to be able to just pass simple structs to store methods without having to construct a complex `Context` object (and instead have that be the responsibility of, say, the resolver)

Reply:
updated! thanks Clay 😄